provisioning/containers/blender-lts/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Original Dockerfile from https://github.com/nytimes/rd-blender-docker/blob/master/dist/2.83-cpu-ubuntu18.04/Dockerfile
+# Includes edits to update to the latest LTS version
+
+# ! With 18.04 there are issues with blender > 2.83.8
+FROM ubuntu:20.04
+
+# Enviorment variables
+ENV DEBIAN_FRONTEND noninteractive
+ENV LC_ALL C.UTF-8
+ENV LANG C.UTF-8
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \ 
+    wget \ 
+    libopenexr-dev \ 
+    bzip2 \ 
+    build-essential \ 
+    zlib1g-dev \ 
+    libxmu-dev \ 
+    libxi-dev \ 
+    libxxf86vm-dev \ 
+    libfontconfig1 \ 
+    libxrender1 \ 
+    libgl1-mesa-glx \ 
+    xz-utils \
+  && apt-get clean -y \
+  && rm -rf /var/lib/apt/lists/*.*
+
+ARG BLENDER_VER="2.83.13-linux64" 
+ENV PATH "$PATH:/bin/2.83/python/bin/"
+ENV BLENDER_PATH "/bin/2.83"
+ENV BLENDERPIP "/bin/2.83/python/bin/pip3"
+ENV BLENDERPY "/bin/2.83/python/bin/python3.7m"
+ENV HW="CPU"
+
+# Download and install Blender
+RUN wget -q https://ftp.nluug.nl/pub/graphics/blender/release/Blender2.83/blender-${BLENDER_VER}.tar.xz \ 
+	&& tar -xf blender-${BLENDER_VER}.tar.xz --strip-components=1 -C /bin \ 
+	&& rm -rf blender-${BLENDER_VER}.tar.xz \ 
+	&& rm -rf blender-${BLENDER_VER} \
+  && rm -rf ${BLENDER_PATH}/python/lib/python3.7/site-packages/numpy
```

Comment:
Nit. The indentation should probably be fixed

Reply:
There were a mix of spaces/tabs, couldn't see it in the editor since it was matching